ui_framework/src/components/icon/index.js

```diff
@@ -1,2 +1,4 @@
 import '!!svg-sprite!./assets/search.svg';
 import '!!svg-sprite!./assets/user.svg';
+import '!!svg-sprite!./assets/apps.svg';
+import '!!svg-sprite!./assets/logo.svg';
```

Comment:
Let's figure out how these are really gonna be done first. This should be automated.